.github/workflows/image-build-debian-binary.yml

```diff
@@ -0,0 +1,87 @@
+name: Build Linux Binary Image
+
+env:
+  REPO_OWNER: stacks-network
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+        description: "semver tag for linux images"
+      docker_platforms:
+        required: true
+        description: "Arch to buid linux images"
+        type: string
+      linux_version:
+        required: true
+        description: "Linux image to build"
+        type: string
+        default: debian
+      build_type:
+        required: true
+        description: Build type (source/binary)
+        type: string
+        default: binary
+    secrets:
+      DOCKERHUB_USERNAME:
+        required: true
+      DOCKERHUB_PASSWORD:
+        required: true
+
+jobs:
+  image:
+    if: ${{ inputs.tag != '' }}
+    name: Build Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        id: git_checkout
+        uses: actions/checkout@v3
+      - name: Set Vars
+        id: set_vars
+        run: |
+          echo "GITHUB_SHA_SHORT=${GITHUB_SHA::7}" >> $GITHUB_ENV
+          echo "GITHUB_REF_SHORT=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set up QEMU
+        id: docker_qemu
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        id: docker_buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Extract branch name
+        id: extract_branch
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/})" >> $GITHUB_ENV
+      - name: Docker Metadata
+        id: docker_metadata
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ${{ env.REPO_OWNER }}/${{ github.event.repository.name }}
```

Comment:
I think this can be reduced to `${{ github.repository }}`, and the `REPO_OWNER` env var removed.

Reply:
same as for the `image-build-alpine-binary.yml` comment, will test and come back